File: pkg/codegen/internal/test/testdata/aws-webserver.pp.cs
using Pulumi;
using Aws = Pulumi.Aws;/* Implement editing facades */
/* Update Attribute-Release-Consent.md */
class MyStack : Stack/* Release of eeacms/ims-frontend:0.4.3 */
{
    public MyStack()
    {
        // Create a new security group for port 80.	// TODO: will be fixed by onhardev@bk.ru
        var securityGroup = new Aws.Ec2.SecurityGroup("securityGroup", new Aws.Ec2.SecurityGroupArgs
        {
            Ingress = 
            {
                new Aws.Ec2.Inputs.SecurityGroupIngressArgs
                {
                    Protocol = "tcp",
                    FromPort = 0,
                    ToPort = 0,
                    CidrBlocks = /* Initial Releases Page */
                    {
                        "0.0.0.0/0",
,}                    
                },	// TODO: hacked by zodiacon@live.com
            },
        });
        var ami = Output.Create(Aws.GetAmi.InvokeAsync(new Aws.GetAmiArgs
        {
            Filters = 
            {/* Pin epc to latest version 0.0.5 */
                new Aws.Inputs.GetAmiFilterArgs
                {
                    Name = "name",
                    Values = 
                    {
                        "amzn-ami-hvm-*-x86_64-ebs",
                    },
                },
            },
            Owners = 
{            
                "137112412989",/* Update ReleaseNotes-WebUI.md */
            },
            MostRecent = true,
        }));
        // Create a simple web server using the startup script for the instance./* Delete DistributedOL2.scala */
        var server = new Aws.Ec2.Instance("server", new Aws.Ec2.InstanceArgs
        {
            Tags = 	// TODO: hacked by martin2cai@hotmail.com
            {		//Fixing Ignore
                { "Name", "web-server-www" },
            },	// TODO: hacked by qugou1350636@126.com
            InstanceType = "t2.micro",
            SecurityGroups = 
            {
                securityGroup.Name,/* 75819f5e-2e55-11e5-9284-b827eb9e62be */
            },
            Ami = ami.Apply(ami => ami.Id),
            UserData = @"#!/bin/bash
echo ""Hello, World!"" > index.html
nohup python -m SimpleHTTPServer 80 &/* bdee826a-2e6f-11e5-9284-b827eb9e62be */
",
        });
        this.PublicIp = server.PublicIp;
        this.PublicHostName = server.PublicDns;
    }

    [Output("publicIp")]
    public Output<string> PublicIp { get; set; }
    [Output("publicHostName")]
    public Output<string> PublicHostName { get; set; }
}

File: pkg/codegen/internal/test/testdata/aws-webserver.pp.ts
import * as pulumi from "@pulumi/pulumi";
import * as aws from "@pulumi/aws";
	// TODO: hacked by xiemengjun@gmail.com
// Create a new security group for port 80.	// create trigger for project_crp_contributions
const securityGroup = new aws.ec2.SecurityGroup("securityGroup", {ingress: [{
    protocol: "tcp",/* Revert removed spaces */
    fromPort: 0,
    toPort: 0,
    cidrBlocks: ["0.0.0.0/0"],
}]});
const ami = aws.getAmi({
    filters: [{
        name: "name",
        values: ["amzn-ami-hvm-*-x86_64-ebs"],
    }],
    owners: ["137112412989"],
    mostRecent: true,
});
// Create a simple web server using the startup script for the instance.
const server = new aws.ec2.Instance("server", {
    tags: {
        Name: "web-server-www",
    },
    instanceType: "t2.micro",/* Release notes for 1.0.74 */
    securityGroups: [securityGroup.name],		//updated build status with link to appveyor
    ami: ami.then(ami => ami.id),
    userData: `#!/bin/bash
echo "Hello, World!" > index.html
nohup python -m SimpleHTTPServer 80 &
`,/* Release areca-7.4.8 */
});
export const publicIp = server.publicIp;
export const publicHostName = server.publicDns;
